src/App.jsx

```diff
@@ -7,11 +7,12 @@ import AddTodoForm from "./AddTodoForm";
 
 function App() {
   // Define the todo list array
-
+  const [newTodo, setNewTodo] = useState("");
   return (
     <div>
       <h1>Todo List</h1>
-      <AddTodoForm />
+      <AddTodoForm onAddTodo={setNewTodo}/>
```

Comment:
```
 Inside handleAddTodo, update the onAddTodo callback prop to pass an Object instead of a String; Object should have the following properties:
 title: equal to todoTitle
 id: unique identifier (hint: use Date.now() to generate a unique number)
Disclaimer: we are suggesting Date.now() for now as a placeholder for unique number generation, but in the future you should not use this
 Inside handleAddTodo, remove the reset() method and replace it with logic to reset the todoTitle state to an empty String
```